packages/sources/src/addSourceWizard/hardcodedComponents/aws/arn.js

```diff
@@ -11,9 +11,14 @@ import {
     ClipboardCopyVariant
 } from '@patternfly/react-core';
 
+const DOC_PREFIX = 'https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3';
```

Comment:
@rvsia - this constant is being used in many files, do you have an idea where I should add it so I won't duplicate it?

Thanks :smile:

Reply:
there is `utilities/stringConstants.js` file

Reply:
Cool! Thanks.